### ISSUE TITLE
Error loading esri.Map.js

### ISSUE THREAD

**Issue by mobotpsmocko** [Esri/esri-loader#200]
### Expected behavior
I am using the esri-loader to integrate esri maps into my Angular 8 application. I followed all of the instructions on the site https://developers.arcgis.com/javascript/latest/guide/angular/. I would expect the map to show.

### Actual behavior

When the application loads I get a 404 error trying to load the esri.Map.js and therefore no map.

### Steps to reproduce the behavior
I followed all of the steps on the developers guide page and got this error. The loader is trying to load https://js.arcgis.com/4.13/dojo/esri.Map.js which is getting a 404 response.

I am using angular 8 node 10 versions. I can supply any other information if needed. Here is my package.json contents

{
  "name": "esri-map-test",
  "version": "0.0.0",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "~8.2.13",
    "@angular/common": "~8.2.13",
    "@angular/compiler": "~8.2.13",
    "@angular/core": "~8.2.13",
    "@angular/forms": "~8.2.13",
    "@angular/platform-browser": "~8.2.13",
    "@angular/platform-browser-dynamic": "~8.2.13",
    "@angular/router": "~8.2.13",
    "core-js": "^2.5.4",
    "esri-loader": "^2.12.0",
    "rxjs": "~6.5.3",
    "tslib": "^1.9.0",
    "zone.js": "~0.9.1"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.803.17",
    "@angular/cli": "~8.3.17",
    "@angular/compiler-cli": "~8.2.13",
    "@angular/language-service": "~8.2.13",
    "@types/arcgis-js-api": "^4.13.0",
    "@types/jasmine": "~2.8.8",
    "@types/jasminewd2": "~2.0.3",
    "@types/node": "~8.9.4",
    "codelyzer": "^5.0.1",
    "jasmine-core": "~2.99.1",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~4.0.0",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "~2.0.1",
    "karma-jasmine": "~1.1.2",
    "karma-jasmine-html-reporter": "^0.2.2",
    "protractor": "~5.4.0",
    "ts-node": "~7.0.0",
    "tslint": "~5.11.0",
    "typescript": "~3.5.3"
  }
}

**Comment by andygup** [comment]
You're getting a `404` because something isn't configured right, the pathname to the ArcGIS API for JavaScript is incorrect. If you are using the default config for esri-loader, everything should just work. 

I recommend checking out our Angular 8 example for `esri-loader`: https://github.com/Esri/angular-cli-esri-map. And, here's a live Angular 7 example: https://stackblitz.com/edit/angular-cli-esri-map2?file=src/main.ts.

**Comment by mobotpsmocko** [comment]
Andy,
I did not change a thing. I just installed the loader, implemented the default code shown on the example and does not work. I forked one of the examples from github and it works so I am puzzled at what is causing the issue. I have taken my prototype and installed the exact versions of the ones in the example (working) and still gets a 404 error but to a different file version. 

In order to implement this into our application, I need to understand what is going wrong. Since I did not change any configuration and followed the documented steps to use the loader, I would expect the same results. New to ESRI so not sure where to look. I do not have any code changing any of the default settings.
